fizzbizz.py

```diff
@@ -0,0 +1,34 @@
+
+
+def fizzbizz(lst):
+    """
+    Simple error checking
+    Takes a list as an argument
+    Numbers that are divisable by 3 are replaced by FIZZ
+    Numbers that are divisable by 5 are replaced by BIZZ
+    Numbers that are divisable by 3 & 5 are replaced by FIZZBIZZ
+    >>> fizzbizz([x for x in range(1,16)])
+    [1, 2, 'FIZZ', 4, 'BIZZ', 'FIZZ', 7, 8, 'FIZZ', 'BIZZ', 11, 'FIZZ', 13, 14, 'FIZZBIZZ']
+    """
+    # Test for a valid lst
+    if not type(lst) is list:
```

Comment:
`isinstance` is a nice feature to use here.  Example:
`if not isinstance(lst, list):`

The advantage of `isinstance` is that you can use it also with classes or tuples that you make along with types.  for example:
```
class Circle():
    pass

my_circle = Circle()
if isinstance(my_circle, Circle)
    print('its a circle')
else:
    print("not a circle")
```